scipy/signal/_peak_finding.py

```diff
@@ -1036,7 +1036,7 @@ def _identify_ridge_lines(matr, max_distances, gap_thresh):
     ----------
     Bioinformatics (2006) 22 (17): 2059-2065.
     :doi:`10.1093/bioinformatics/btl355`
-    http://bioinformatics.oxfordjournals.org/content/22/17/2059.long
+    https://academic.oup.com/bioinformatics/article/22/17/2059/274284
```

Comment:
Actually, it's better to remove the https:// link. The DOI link alone is enough, and will go to the same place.

Reply:
Thanks for the advice. I have updated this entry to be in all three places and removed the link (given the DOI link exists)